app/controllers/concerns/europeana/catalog.rb

```diff
@@ -101,6 +104,26 @@ def more_like_this(document, field = nil, extra_controller_params = {})
       search_results(mlt_params, search_params_logic)
     end
 
+    def media_mime_type(document)
```

Comment:
Assignment Branch Condition size for media_mime_type is too high. [21.1/15]<br>Method has too many lines. [14/10]